.github/workflows/ci-dgraph-code-coverage.yml

```diff
@@ -0,0 +1,74 @@
+name: ci-dgraph-code-coverage
+on:
+  push:
+    branches:
+      - main
```

Comment:
you can enable coverage code (I think), you can introduce the `release/*` inside `push` & `pull_request`